src/main/java/org/fundacionjala/coding/jimmy/Kata1.java

```diff
@@ -8,7 +8,7 @@
  */
 public class Kata1 {
 
-    private static final int THRESHOLD = 5;
+      private static final int THRESHOLD = 5;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)